core/jazz_ui/src/app/pages/environment-assets/env-assets-section.component.html

```diff
@@ -47,21 +47,57 @@
+                                </span>
+                                <span *ngSwitchCase='"s3"' class="icon-icon-S3">
+                                    <span class="path1"></span><span class="path2"></span><span class="path3"></span><span class="path4"></span><span class="path5"></span><span class="path6"></span><span class="path7"></span><span class="path8"></span><span class="path9"></span><span class="path10"></span><span class="path11"></span><span class="path12"></span>
+                                </span>
+                                <span *ngSwitchCase='"ec2"' class="icon-icon-EC2">
+                                    <span class="path1"></span><span class="path2"></span><span class="path3"></span><span class="path4"></span><span class="path5"></span><span class="path6"></span><span class="path7"></span><span class="path8"></span>
+                                </span>
+                                <span *ngSwitchCase='"eventhubs_eventhub"'>
+                                    <img class='server-icon' src='./../../../assets/images/Event_Hubs.svg'>
+                                </span>
+                                <span *ngSwitchCase='"functionapp"'>
+                                    <img class='server-icon' src='./../../../assets/images/Function_Apps.svg'>
+                                </span>
+                                <span *ngSwitchCase='"storage_account"'>
+                                    <img class='server-icon' src='./../../../assets/images/Storage_Accounts.svg'>
+                                </span>
+                                <span *ngSwitchCase='"cosmosdb_database"'>
+                                    <img class='server-icon' src='./../../../assets/images/Azure_Cosmos_DB.svg'>
+                                </span>
+                                <span *ngSwitchCase='"cosmosdb_account"'>
+                                    <img class='server-icon' src='./../../../assets/images/Azure_Database.svg'>
+                                </span>
+                                <span *ngSwitchCase='"storage_blob_container"'>
+                                    <img class='server-icon' src='./../../../assets/images/Blob_Storage.svg'>
+                                </span>
+                                <span *ngSwitchCase='"cosmosdb_collection"'>
+                                    <img class='server-icon' src='./../../../assets/images/Virtual_Clusters.svg'>
+                                </span>
+                                <span *ngSwitchCase='"eventhubs_namespace"'>
+                                    <img class='server-icon' src='./../../../assets/images/Event_Grid_Domains.svg'>
```

Comment:
Let's follow a convention on how we name these files and their locations. Suggestion -> `../assets/images/azure/cosmos_db.svg`, `../assets/images/aws/lambda.svg`, `../assets/images/azure/service_bus_relay.svg` etc.

Reply:
Let's have a separate story to re-organize these assets.